packages/sdk-components-react-radix/src/input.ws.ts

```diff
@@ -0,0 +1,79 @@
+import { FormTextFieldIcon } from "@webstudio-is/icons/svg";
+import {
+  defaultStates,
+  type PresetStyle,
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import { input } from "@webstudio-is/react-sdk/css-normalize";
+import { props } from "./__generated__/input.props";
+import * as tc from "./theme/tailwind-classes";
+
+const presetStyle = {
+  input,
+} satisfies PresetStyle<"input">;
+
+export const meta: WsComponentMeta = {
+  category: "radix",
+  type: "control",
+  invalidAncestors: ["Button"],
+  label: "Input",
+  icon: FormTextFieldIcon,
+  presetStyle,
+  states: [
+    ...defaultStates,
+    { selector: ":disabled", label: "Disabled" },
+    { selector: "::placeholder", label: "Placeholder" },
+    { selector: "::file-selector-button", label: "File Selector Button" },
+    { selector: ":valid", label: "Valid" },
+    { selector: ":invalid", label: "Invalid" },
+    { selector: ":required", label: "Required" },
+    { selector: ":optional", label: "Optional" },
+    { selector: ":disabled", label: "Disabled" },
+    { selector: ":enabled", label: "Enabled" },
+    { selector: ":read-only", label: "Read Only" },
+    { selector: ":read-write", label: "Read Write" },
```

Comment:
We need to add  a submenu, @taylornowotny was supposed to provide some design in figma

- [ ] We discussed with @taylornowotny  to reduce the list for both input components, because its too much at glance and these are very rare
https://github.com/webstudio-is/webstudio-builder/issues/2029
- [ ]  For now I would comment these additional once out and make sure we reuse this list. This way once we have a submenu, it comes to both.

Reply:
commented for now

Reply:
we need the same on the other input why can't we reuse these states?

Reply:
Added comment